File: AreaTriangulo.rb
#Declarar Variables y otros
imprimir  "Ejercicios 01: Area de un Triangulo"
#Datos de entrada
pone "Ingrese Base:"
b = obtiene . chomp . to_i
pone "Ingrese Altura:"
h = obtiene . chomp . to_i
#Proceso
área = ( b * h ) / 2
#Datos de Salida
pone  "El area del Triangulo es:" ,  area

File: distint-language/AreaTriangulo.java
import java.util.Scanner;

class AreaTriangulo{ 
Scanner teclado=new Scanner(System.in);
public static void main(String [] arg){
 //Definir Variables y otros
  System.out.println("Hola Mundo");
  int b, h, area;
  //Datos de Entrada
  System,out,orintln("Ingrese la Base");
  b=teclado.nextInt();
  System,out,orintln("Ingrese la Altura");
  h=teclado.nextInt();
  //Proceso
  area=(b*h)/2;
  //Datos de Salida
  System,out,orintln("Area de un Triangulo es:"+area);
}
}